### PR TITLE
refactor: change column name run in units to flowcell.

### PR DESCRIPTION
**Pull request by Smeds**
### Review and tests: 
- [ ] Tests pass

### CODE REVIEW

File: hydra_genetics/commands/create.py [hunk 1]
```diff
@@ -360,7 +360,7 @@ def init(self):
                 log.warn("File exists {} overwriting!!!".format(units_file_name))
         with open(units_file_name, "w") as output:
             output.write("\t".join(["sample", "type", "platform", "barcode", "machine",
-                                    "run", "lane", "fastq1", "fastq2", "adapter"]))
+                                    "", "lane", "fastq1", "fastq2", "adapter"]))
```
Comment (marrip) [inline hydra_genetics/commands/create.py:363]: Why is it empty string?

File: hydra_genetics/commands/create.py [hunk 2]
```diff
@@ -392,7 +392,7 @@ def extract_run_information(file_path, number_of_reads=200, every_n_reads=1000,
     :type number_of_reads: integer
     :param: warning_threshold: raise a warning for char with lower occurences this value
     :type warning_threshold: float
-    :param compare_first_and_last_read: compare first read with last read to detect merged lanes or runs
+    :param compare_first_and_last_read: compare first read with last read to detect merged lanes or s
```
Comment (marrip) [inline hydra_genetics/commands/create.py:395]: What does `s` stand for?

File: hydra_genetics/commands/create.py (hunk 1 above)
Comment (Smeds) [reply hydra_genetics/commands/create.py:363]: typo

Comment (Smeds) [reply hydra_genetics/commands/create.py]: fixed

File: hydra_genetics/commands/create.py (hunk 2 above)
Comment (Smeds) [reply hydra_genetics/commands/create.py]: I did a replace one "run" and must have accidentally replaced it with a empty string instead of flowcell

Comment (Smeds) [reply hydra_genetics/commands/create.py]: fixed now

**Review by marrip — APPROVED** [review]

**Pull request merged**